core/src/main/scala/kafka/server/KafkaRequestHandler.scala

```diff
@@ -467,6 +480,12 @@ class BrokerTopicAggregatedMetric() {
     partitionMetricValues.put(partition, partitionValue)
   }
 
+  def incrementPartitionMetricValue(partition: Int): Unit = partitionMetricValues.compute(partition, (_, value) => value + 1)
```

Comment:
Can we remove this method as it is not used anymore?

Reply:
Ah, right, just updated.